deploy/compose/docker-compose.yml

```diff
@@ -0,0 +1,301 @@
+services:
+  # TLS cert for tink-server and docker registry
+  tls-gen:
+    image: cfssl/cfssl
+    entrypoint: /code/tls/generate.sh
+    command: ["$TINKERBELL_HOST_IP"]
+    environment:
+      FACILITY: ${FACILITY:-onprem}
+    volumes:
+      - ${REPO_TOP_LEVEL:-.}:/code
+      - certs:/certs/${FACILITY:-onprem}:rw
+
+  # User creds for the registry
+  registry-auth:
+    image: httpd:2
+    entrypoint: htpasswd
+    working_dir: /auth
+    command:
+      [
+        "-Bbc",
+        ".htpasswd",
+        "${TINKERBELL_REGISTRY_USERNAME:-admin}",
+        "${TINKERBELL_REGISTRY_PASSWORD:-Admin1234}",
+      ]
+    volumes:
+      - auth:/auth:rw
```

Comment:
Not directly related to this PR, but I would love to see us remove the need for the registry in the sandbox, at least as a requirement.

Reply:
Me too. I know there has been a good amount of discussion in this [issue](https://github.com/tinkerbell/tink/issues/469) about this idea.

Reply:
Would removing the registry in the sandbox make it still possible to run it in an airgapped environment (so no access to quay.io)?

Reply:
The idea would be to still support airgapped environments, but not to deploy the sandbox that way by default.